docs/release/0.14.0.rst

```diff
@@ -0,0 +1,56 @@
+============================
```

Comment:
This top line seems to be produced as-is in git-flavoured markdown.

Reply:
I removed this line.